src/cryptonote_core/blockchain.cpp

```diff
@@ -2028,9 +2028,9 @@ bool Blockchain::get_output_distribution(uint64_t amount, uint64_t from_height,
   {
     switch (m_nettype)
     {
-      case STAGENET: start_height = stagenet_hard_forks[3].height; break;
-      case TESTNET: start_height = testnet_hard_forks[3].height; break;
-      case MAINNET: start_height = mainnet_hard_forks[3].height; break;
+      case STAGENET: start_height = stagenet_hard_forks[0].height; break;
+      case TESTNET: start_height = testnet_hard_forks[0].height; break;
+      case MAINNET: start_height = mainnet_hard_forks[0].height; break;
       default: return false;
```

Comment:
We have rct outputs from start of chain, also 3 indexes out of bounds. Only have v7, v8 entries right now.